docs/source/cookbook/optical_mosaic.rst

```diff
@@ -1,10 +1,564 @@
+    
+    .. line-break::
+
+-   :guilabel:`A+B`: `Sentinel-2 Multispectral instrument <https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S2>`__ is a wide-swath, high-resolution, multi-spectral imaging mission supporting Copernicus Land Monitoring studies, including the monitoring of vegetation, soil and water cover, as well as observation of inland waterways and coastal areas.
+
+.. thumbnail:: ../img/cookbook/optical_mosaic/select_source.png
+    :title: The source selection panel
+    :group: optical-mosaic-recipe
+
+To validate your selection, click on :icon:`fa fa-check` :guilabel:`Apply` button.
+
+Scenes
+^^^^^^
+
+.. warning:: 
+
+    If Sentinel and Landsat data have been selected, you will be forced to use all scenes. As the tilling system from Sentinel and Landsat data are different, it's impossible to select scenes using the tool presented in the next sections.
+
+You can use multiple options to select the best scenes for your mosaic. The most simple is to use every image available based on the date parameters. Click :guilabel:`use all scenes` and all the images will be integrated into the mosaic. 
+
+Select :guilabel:`select scenes` and 3 new selection options will become available. SEPAL is sorting the images available for each tile, three :code:`priority` options are available, choose the one that suits your analysis: 
+
+-   :guilabel:`cloud free`: will give priority to images with zero or few clouds. 
+-   :guilabel:`target date`: will give priority to images that match with the target date 
+-   :guilabel:`balanced`: will give priority to images that maximize both cloud and target date.
+
+To validate your selection, click on :icon:`fa fa-check` :guilabel:`Apply` button.
+
+.. thumbnail:: ../img/cookbook/optical_mosaic/scene_method.png
+    :title: The source selection panel
+    :group: optical-mosaic-recipe
+
+Composite
+^^^^^^^^^
+
+.. note:: 
+
+    This step is optional, SEPAL provides the folowing options by default: 
+
+    -   **Correction**: :guilabel:`SR`, :guilabel:`BRDF`
+    -   **Pixel filters**: no filters
+    -   **Cloud detection**: :guilabel:`QA bands`, :guilabel:`Cloud score`
+    -   **Cloud masking**: :guilabel:`Moderate`
+    -   **Cloud buffering**: :guilabel:`None`
+    -   **Snow masking**: :guilabel:`On`
+    -   **Composing method**: :guilabel:`medoid`
+
+To create a mosaic, you will need to provide SEPAL with the compositing method to create the final image. Here is a description of all the possible compositing options available. 
```

Comment:
I'm not sure if you missed something after this paragraph.

Reply:
I'm describing it in the next sub-sections, maybe I should replace it with "You'll find in the next sections the description of all the available composition options"